src/lbcamden/components/guide-header/_index.scss

```diff
@@ -35,7 +41,8 @@
     }
 
     @include govuk-media-query($from: desktop) {
-      width: calc(100% / 3);
+      // 90px is the width of the three padding-right values
+      width: calc((100% - 90px) / 3);
 
```

Comment:
The comment is helpful, but might be better to avoid the hardcoded pixel value alltogether and use `govuk-spacing(6) * 3` instead?